projects/assets/src/translations/en/common.ts

```diff
@@ -171,6 +171,8 @@ export const common = {
     required: 'This field is required',
     cxInvalidEmail: 'This is not a valid email format',
     cxInvalidPassword: 'This is not a valid password format',
+    cxPasswordRequirements:
+      'Passwords must include: both uppercase and lowercase letters, one or more numerical digits, one or more special characters, at least 8 total characters', // change me to reflect the current password requirements
```

Comment:
Thanks for working on this, it's a really important change!

I think once you get the password requirements down, you can sync with @gladius-mtl to make sure that the wording for the warning message is fine. (Hope you're the right person to check this out Matthew haha)

Reply:
I propose the following text:

`Passwords must include at least 8 characters, a mixture of both uppercase and lowercase letters, at least one number, and at least one special character, such as !@#?$`

Reply:
Password requirement message changed accordingly!